### PR TITLE
Show recently processed jobs

### PR DESCRIPTION
**Pull request by eagletmt**
I'd like to monitor which job execution is spiking when Barbeque becomes busy.

@cookpad/dev-infra please review

### CODE REVIEW

File: app/views/barbeque/monitors/index.html.haml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+              - jobs = @recently_processed_jobs[date_hour]
+              - jobs.keys.sort.each do |job_id|
+                - job = jobs[job_id]
+                %tr
+                  %td= date_hour
+                  %td= link_to(job[:app_name], app_path(job[:app_id]))
+                  %td= link_to(job[:job_name], job_definition_path(job[:job_id]))
+                  %td= job[:count]
+
+:coffee
```
Comment (hogelog) [inline app/views/barbeque/monitors/index.html.haml]: Use camelCase to variable name https://github.com/cookpad/styleguide/blob/master/coffeescript.ja.md#変数名

File: app/views/barbeque/monitors/index.html.haml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  chart_div = document.getElementById('recently-processed-jobs-chart')
+  recently_processed_jobs = JSON.parse(chart_div.dataset.jobs)
+
+  counts_per_job = {}
+  for own date_hour, jobs of recently_processed_jobs
+    console.log(jobs)
```
Comment (hogelog) [inline app/views/barbeque/monitors/index.html.haml]: debug log?

Comment (eagletmt) [inline app/views/barbeque/monitors/index.html.haml]: 🙈 https://github.com/cookpad/barbeque/pull/19/commits/894c5bfb4b6103a39404a6f1721918d4d1125563

File: app/views/barbeque/monitors/index.html.haml (hunk 1 above)
Comment (eagletmt) [inline app/views/barbeque/monitors/index.html.haml]: Fixed in https://github.com/cookpad/barbeque/pull/19/commits/1b0e2cf3934f28733a273169864d9a52e8696ea8

**Pull request merged**